regression/goto-analyzer/value-set-function-pointers/test.desc

```diff
@@ -0,0 +1,23 @@
+CORE
+main.c
+--variable-sensitivity --vsd-structs every-field --vsd-arrays every-element --vsd-pointers value-set --vsd-values set-of-constants --show --pointer-check --three-way-merge
```

Comment:
Right - I see what you have done.  For the sake of a cleaner PR it might be good if the reduced one is the new one but ... it's not that big a deal.

Reply:
Done!